drivers/fastboot/fb_command.c

```diff
@@ -86,6 +86,15 @@ static const struct {
 		.dispatch = oem_format,
 	},
 #endif
+	/* Add command to lock/unlock emmc bootloader block access */
+	[FASTBOOT_COMMAND_LOCK_CRITICAL] =  {
+		.command = "flashing lock_critical",
+		.dispatch = fastboot_lock_critical
+	},
+	[FASTBOOT_COMMAND_UNLOCK_CRITICAL] =  {
+		.command = "flashing unlock_critical",
+		.dispatch = fastboot_unlock_critical
+	},
```

Comment:
We need to make sure these functions are only available to devices that support it (right now only Nitrogen8M).
Can you add a flag around it, for all our i.MX6 platforms, since U-Boot is in NOR those won't work.

Reply:
this function isn't necessarily uboot specific (though we are using it for that).  Generically, this just unlocks the emmc boot partition, and that may be useful regardless of whether that boot partition is used for uboot or not.   The implementation of the code is featurized for emmc, so leaving this as is should be fine.